src/python/grpcio_reflection/grpc_reflection/v1alpha/reflection.py

```diff
@@ -120,6 +118,7 @@ def _list_services(self):
             ]))
 
     def ServerReflectionInfo(self, request_iterator, context):
+        #pylint: disable=unused-argument
```

Comment:
Does this actually have effect here? I thought it had to be at the end of the previous line.